src/cbc_sdk/platform/processes.py

```diff
@@ -683,14 +760,19 @@ def _get_query_parameters(self):
         query = self._query_builder._collapse()
         if self._query_builder._process_guid is not None:
             args["process_guid"] = self._query_builder._process_guid
-        elif self._model_unique_id is not None:
-            args["process_guid"] = self._model_unique_id
+        elif 'process_guid:' in query:
+            q = query.split('process_guid:', 1)[1].split(' ', 1)[0]
+            args["process_guid"] = q
+
         if 'parent_guid:' in query:
             # extract parent_guid from where() clause
             parent_guid = query.split('parent_guid:', 1)[1].split(' ', 1)[0]
             args["parent_guid"] = parent_guid
         return args
 
+    def _count(self):
+        raise ApiError('The result is not iterable')
```

Comment:
As much as this is the case the SummaryQuery class inherits IterableQueryMixin so this message is confusing. We should consider removing the IterableQueryMixin inheritance as well.